adafruit_platformdetect/chip.py

```diff
@@ -211,6 +211,9 @@ def _linux_id(self) -> Optional[str]:
         if self.detector.check_dt_compatible_value("st,stm32mp157"):
             return chips.STM32MP157
 
+        if self.detector.check_dt_compatible_value("st,stm32mp153"):
```

Comment:
I notice you are looking for a string with stm32mp153, but are defining it as an stm32mp157. If yours is specifically a 153, please use that. If not, was this a typo?

Reply:
the exact value located in the file /proc/device-tree/compatible is:
`olimex,stm32mp1xx-olinuxino-lime^@st,stm32mp153^@`
since the branding on the chip itself says it´s an stm32mp157DAA i have taken the one above.

Reply:
Ok, this makes sense. Thanks.